pkgs/development/python-modules/sshpubkeys/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, cryptography
+, ecdsa
+}:
+
+buildPythonPackage rec {
+  pname = "sshpubkeys";
+  version = "3.1.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "b388399caeeccdc145f06fd0d2665eeecc545385c60b55c282a15a022215af80";
+  };
+
+  buildInputs = [ cryptography ecdsa ];
+
+  # For some reason nix can't find the "tests" module.
+  doCheck = false;
```

Comment:
Seems somethings not right with the pypi tarball.
Everything works fine when I use the github repo.

```nix
src = fetchFromGitHub {
    owner = "ojarva";
    repo = "python-sshpubkeys";
    rev = "v${version}";
    sha256 = "0000000000000000000000000000000000000000000000000000"; # change this
};
```

Reply:
Please also add a comment above it explaining why we have to use github.